modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/exception/mapper/PrincipalExceptionMapper.java

```diff
@@ -44,7 +44,7 @@ protected Problem getProblem(PrincipalException principalException) {
 			status = Response.Status.NOT_FOUND;
 		}
 
-		return new Problem(status, null);
+		return new Problem(status, principalException.getMessage());
```

Comment:
I'm not sure if we were avoiding sending the exception message here for some reason, but I felt it is more correct to give more information to the API consumer